### PR TITLE
add 1-month Optim Bond tokens to Liqwid TVL

### PR DESCRIPTION
**Pull request by kylixafonso**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by kylixafonso** [comment]
Note: The Optim Bond tokens and their value are still not being counted.

**Comment by llamatester** [comment]
The adapter at projects/liqwid exports TVL: 
        
 
 
    cardano-borrowed          12.25 M
    cardano                   38.09 M
    cardano-staking           6.90 M
    borrowed                  12.25 M
    staking                   6.90 M
    
    total                    38.09 M

File: projects/liqwid/index.js [hunk 1]
```diff
@@ -67,16 +92,36 @@ const tokenMapping = {
...
+  }
+  const loans = await getLoans(0)
+  const relevantLoans =
+    loans.filter(l => (l.collaterals.filter(c => c.id === "OptimBond1")).length != 0)
+  const bonds =
+    relevantLoans.map(l => l.collaterals[0].amount).reduce((acc, amount) =>
+      acc + Number(amount), 0)
+  const { Market: { price: { price: adaPrice } } } = await graphQuery(endpoint, queryAdaPrice)
+
+  return bonds * 100 * adaPrice
```
Comment (waynebruce0x) [inline projects/liqwid/index.js:112]: If you remove now export just the quantity of bonds, rather than priced in USD, it should work

**Comment by llamatester** [comment]
The adapter at projects/liqwid exports TVL: 
        
 
 
    cardano-borrowed          12.15 M
    cardano                   151.56 M
    cardano-staking           6.78 M
    borrowed                  12.15 M
    staking                   6.78 M
    
    total                    151.56 M

File: projects/liqwid/index.js (hunk 1 above)
Comment (kylixafonso) [reply projects/liqwid/index.js]: Trying now

**Comment by llamatester** [comment]
The adapter at projects/liqwid exports TVL: 
        
 
 
    cardano-borrowed          12.35 M
    cardano                   38.71 M
    cardano-staking           6.79 M
    borrowed                  12.35 M
    staking                   6.79 M
    
    total                    38.71 M

**Comment by kylixafonso** [comment]
Seems like it might be pricing Optim bond tokens wrongly? @waynebruce0x

**Comment by kylixafonso** [comment]
Should be ready for merge.

**Comment by llamatester** [comment]
The adapter at projects/liqwid exports TVL: 
        
 
 
    cardano-borrowed          12.37 M
    cardano                   39.06 M
    cardano-staking           6.53 M
    borrowed                  12.37 M
    staking                   6.53 M
    
    total                    39.06 M

**Pull request merged**